src/field_control_avatar.c

```diff
@@ -613,6 +614,13 @@ static bool8 TryStartStepCountScript(u16 metatileBehavior)
             ScriptContext_SetupScript(MossdeepCity_SpaceCenter_2F_EventScript_RivalRayquazaCall);
             return TRUE;
         }
+#if I_VS_SEEKER_CHARGING != 0
+        if (UpdateVsSeekerStepCounter() == TRUE)
+        {
+            ScriptContext_SetupScript(EventScript_VsSeekerChargingDone);
+            return TRUE;
+        }
+#endif
```

Comment:
```suggestion
        if (I_VS_SEEKER_CHARGING != 0 && UpdateVsSeekerStepCounter() == TRUE)
        {
            ScriptContext_SetupScript(EventScript_VsSeekerChargingDone);
            return TRUE;
        }
```

Reply:
Fixed in https://github.com/PokemonSanFran/pokeemerald-expansion/commit/e4fe0e19eff65df5e93af055551d598e7ff4fcb8